dashboards/vm/victorialogs-cluster.json

```
@@ -3414,38 +7907,55 @@
             "tooltip": {
               "hideZeros": false,
               "mode": "multi",
-              "sort": "none"
+              "sort": "desc"
             }
           },
-          "pluginVersion": "11.5.0",
+          "pluginVersion": "12.0.2",
           "targets": [
             {
               "datasource": {
-                "type": "victoriametrics-metrics-datasource",
+                "type": "prometheus",
                 "uid": "$ds"
               },
               "editorMode": "code",
-              "expr": "sum(max_over_time(process_num_threads{job=~\"$job\", instance=~\"$instance\"}[$__rate_interval])) by(job)",
+              "exemplar": false,
+              "expr": "max(process_cpu_cores_available{job=~\"$job\", instance=~\"$instance_select\"})",
```

```suggestion
              "expr": "max(process_cpu_cores_available{job=~\"$job\", instance=~\"$instance\"})",
```

And other ocurrences

---

I'm not sure I understand this change. Could you elaborate?

---

Oh, my fault - I imported the dashboard, but somehow `instance_select` didn't appear (`instance` works though)